app/models/organization.rb

```diff
@@ -8,10 +8,13 @@ def self.default_scope
   has_many :orders
   has_many :open_orders, -> { where(status: Order.open_statuses) }, class_name: "Order"
   has_many :approved_orders, -> { for_approved_statuses.order(order_date: :desc) }, class_name: "Order"
+  has_many :organization_programs
+  has_many :programs, through: :organization_programs
   has_many :organization_addresses
   has_many :addresses, through: :organization_addresses
   accepts_nested_attributes_for :addresses, allow_destroy: true
   validates :name, uniqueness: true
+  validates :programs, length: { minimum: 1 }
```

Comment:
just verifying we confirmed this with Lisa

Reply:
Indeed, it was requested by her :-)